sources/europe/gb/OSOpenMapLocal.geojson

```diff
@@ -1,12 +1,12 @@
 {
     "type": "Feature",
     "properties": {
-        "id": "OS-OpenMap_Local_2022_10",
-        "name": "OS OpenMap Local (October 2022)",
+        "id": "OS-OpenMap_Local_2023_044",
```

Comment:
Should the “044” be “04”, as I think this is the reference to the month.

Reply:
or drop any "date" reference ?